src/analyses/base.ml

```diff
@@ -546,7 +544,8 @@ struct
     done;
     (* Return the list of elements that have been visited. *)
     if M.tracing then M.traceu "reachability" "All reachable vars: %a\n" AD.pretty !visited;
-    List.map AD.singleton (AD.elements !visited)
+     List.of_seq @@ (Seq.map AD.singleton) @@ List.to_seq (AD.elements !visited)
```

Comment:
Since there is only operation (`List.map`) that is replaced, can one hope that this change actually can have a positive performance impact?